common/src/main/java/bisq/common/util/BitcoinUtils.java

```diff
@@ -0,0 +1,58 @@
+package bisq.common.util;
```

Comment:
Not sure about this one, I thought about it as a generic way to identifying wallet address and tx id to be reusable. If I were to implement a validator, the validator would be calling those methods as well.

 Doesn't it make more sense as is, as it could be used in many areas to make sure a wallet address / transaction is valid?

Reply:
As we have a validation package and the class is doing validation it would be for me logical to look up for that class there.
In Utils we usually dont have validation code but util-specific stuff. Validation is a general concept worth to be grouped apart from rather unspecific utils.

Reply:
There have been a refacotring PR in the meantime for common to clean up the utils package. Be sure to have rebased on main